query/src/servers/http/clickhouse_handler.rs

```diff
@@ -306,6 +306,11 @@ pub async fn clickhouse_handler_post(
     let default_format = get_default_format(&params, headers).map_err(BadRequest)?;
     let mut sql = params.query();
     sql.push_str(body.into_string().await?.as_str());
+    // other parts of the request already logged in middleware
+    tracing::info!(
+        "receive clickhouse http post, (query + body)[0..100] = {}",
+        &sql[0..100]
```

Comment:
What happens if the user inputs a SQL with only 10 bytes?

Reply:
my fault, fixed,   thought it is python..